src/librustdoc/json/mod.rs

```diff
@@ -38,8 +38,8 @@ crate struct JsonRenderer<'tcx> {
     cache: Rc<Cache>,
 }
 
-impl JsonRenderer<'tcx> {
-    fn sess(&self) -> &'tcx Session {
+impl JsonRenderer<'_> {
+    fn sess(&self) -> &'_ Session {
```

Comment:
I think this should be

```suggestion
impl<'tcx> JsonRenderer<'tcx> {
    fn sess(&self) -> &'tcx Session {
```

Reply:
I can change it. Code currently compiles, since it seems no code cares about that returned lifetime being shorter than `'tcx`.

Reply:
Yeah, but it can cause problems in the future. It's better for the lifetimes to be correct.